### PR TITLE
Add stubs under `Music_theory`

### PR DESCRIPTION
**Pull request by Gamingzone123**
ticks off some points in #3151 
also moved some music theory articles to actually be under music theory

also, don't know what the checklist means by 'layers'

- [x] add redirects in `redirect.yaml`

### CODE REVIEW

**Comment by Walavouchey** [comment]
you've accidentally committed to `.gitignore` in [0b945ac](https://github.com/ppy/osu-wiki/pull/6995/commits/0b945acaa19bb6f74831db8bd8431b7911f62dd4)

File: wiki/Music_theory/Measure/en.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+---
+
+# Measure
+
+A **measure** (or bar) in music refers to a single unit of time featuring a specific number of [beats](/wiki/Music_theory/Beat) played at a particular [tempo](/wiki/Music_theory/Tempo).
+
+![screenshot of a measure in timing tab](img/measure_timing.png "All white ticks represent beats, and the larger ones represent the start of a new measure")
+![screenshot of a measure in compose tab](img/measure_compose.png "White ticks represent the start of a new measure and the coloured ticks represent each beat within said measure")
+
+screenshots taken of the [beatmap editor timeline](/wiki/Client/Beatmap_editor/Timelines)
```
Comment (Walavouchey) [inline wiki/Music_theory/Measure/en.md:12]: the coloured ticks aren't beats, they're subdivisions of a beat

```suggestion
A **measure** (or **bar**) in music theory refers to a single unit of time featuring a specific number of [beats](/wiki/Music_theory/Beat) played at a particular [tempo](/wiki/Music_theory/Tempo).

In the [beatmap editor timeline](/wiki/Client/Beatmap_editor/Timelines), a measure can be identified as the gap between two large white ticks:

![A measure in the timing tab timeline](img/measure_timing.png "White ticks represent beats, with the larger ones representing the start of a new measure.")

![A measure in the compose tab timeline](img/measure_compose.png "White ticks represent beats, with the larger ones representing the start of a new measure")
```

File: wiki/Music_theory/Measure/en.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+stub:true
```
Comment (Walavouchey) [inline wiki/Music_theory/Measure/en.md:2]:
```suggestion
stub: true
tags:
  - bar
```

File: wiki/Music_theory/Rhythm/en.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+stub:true
```
Comment (Walavouchey) [inline wiki/Music_theory/Rhythm/en.md:2]:
```suggestion
stub: true
```

File: wiki/Music_theory/Rhythm/en.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+---
+
+# Rhythm
+
+**Rhythm** is the pattern of sound, silence, and emphasis in a song. In music theory, rhythm refers to the recurrence of notes and rests (silences) in time. When a series of notes and rests repeats, it forms a rhythmic pattern. In addition to indicating when notes are played, musical rhythm also stipulates how long they are played and with what intensity. This creates different note durations and different types of accents.[[1]][r] In osu, notes are represented by [beats](/wiki/Music_theory/Beat), accents are represented by [hitsounds](/wiki/Beatmapping/Hitsound)
+
+#### References
+1. [Understanding rhythm in music - masterclass](https://www.masterclass.com/articles/understanding-rhythm-in-music)
```
Comment (Walavouchey) [inline wiki/Music_theory/Rhythm/en.md]:
```suggestion
## References

1. [Understanding rhythm in music - masterclass](https://www.masterclass.com/articles/understanding-rhythm-in-music)

[r]: #references
```

File: wiki/Music_theory/Syncopation/en.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
+stub:true
```
Comment (Walavouchey) [inline wiki/Music_theory/Syncopation/en.md:2]:
```suggestion
stub: true
```

File: wiki/Music_theory/Rhythm/en.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+---
+stub:true
+---
+
+# Rhythm
+
+**Rhythm** is the pattern of sound, silence, and emphasis in a song. In music theory, rhythm refers to the recurrence of notes and rests (silences) in time. When a series of notes and rests repeats, it forms a rhythmic pattern. In addition to indicating when notes are played, musical rhythm also stipulates how long they are played and with what intensity. This creates different note durations and different types of accents.[[1]][r] In osu, notes are represented by [beats](/wiki/Music_theory/Beat), accents are represented by [hitsounds](/wiki/Beatmapping/Hitsound)
```
Comment (Walavouchey) [inline wiki/Music_theory/Rhythm/en.md]:
```suggestion
**Rhythm** is the pattern of sound, silence, and emphasis in a song. In music theory, rhythm refers to the recurrence of notes and rests (silences) in time. When a series of notes and rests repeats, it forms a rhythmic pattern. In addition to indicating when notes are played, musical rhythm also stipulates how long they are played and with what intensity. This creates different note durations and different types of accents. [[1]][r] In osu, notes are represented by [beats](/wiki/Music_theory/Beat), accents are represented by [hitsounds](/wiki/Beatmapping/Hitsound)
```

File: wiki/Music_theory/Tempo/en.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+---
+stub:true
```
Comment (Walavouchey) [inline wiki/Music_theory/Tempo/en.md:2]:
```suggestion
stub: true
```

File: wiki/Music_theory/Tempo/en.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+---
+
+# Tempo
+
+**Tempo** is the speed at which a piece of music is played and is communicated through the use of BPM (or Beats Per Miinute).
```
Comment (Walavouchey) [inline wiki/Music_theory/Tempo/en.md:7]:
```suggestion
**Tempo** is the speed at which a piece of music is played and is communicated, measured in beats per minute (BPM).
```

File: wiki/Music_theory/Tempo/en.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+
+[Screenshot of BPM on the beatmap information page](img/BPM_beatmap_info.png "The BPM of a song is shown on a beatmaps information page next to the metronome symbol")
```
Comment (Walavouchey) [inline wiki/Music_theory/Tempo/en.md:9]:
```suggestion
[Screenshot of BPM on the beatmap information page](img/BPM_beatmap_info.png "The BPM of a song is shown on a beatmap's information page next to the metronome symbol")
```

File: wiki/Music_theory/Rhythm/en.md [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+stub:true
+---
+
+# Rhythm
+
+**Rhythm** is the pattern of sound, silence, and emphasis in a song. In music theory, rhythm refers to the recurrence of notes and rests (silences) in time. When a series of notes and rests repeats, it forms a rhythmic pattern. In addition to indicating when notes are played, musical rhythm also stipulates how long they are played and with what intensity. This creates different note durations and different types of accents.[^1] In osu, notes are represented by [beats](/wiki/Music_theory/Beat), accents are represented by [hitsounds](/wiki/Beatmapping/Hitsound)
+
+## References
+
+[^1]: [Understanding rhythm in music - masterclass](https://www.masterclass.com/articles/understanding-rhythm-in-music)
```
Comment (Walavouchey) [inline wiki/Music_theory/Rhythm/en.md:11]:
```suggestion
## References

1. [Understanding rhythm in music - masterclass](https://www.masterclass.com/articles/understanding-rhythm-in-music)

[r]: #references
```

File: wiki/Music_theory/Rhythm/en.md [hunk 11]
```diff
@@ -0,0 +1,11 @@
+---
+stub:true
+---
+
+# Rhythm
+
+**Rhythm** is the pattern of sound, silence, and emphasis in a song. In music theory, rhythm refers to the recurrence of notes and rests (silences) in time. When a series of notes and rests repeats, it forms a rhythmic pattern. In addition to indicating when notes are played, musical rhythm also stipulates how long they are played and with what intensity. This creates different note durations and different types of accents.[^1] In osu, notes are represented by [beats](/wiki/Music_theory/Beat), accents are represented by [hitsounds](/wiki/Beatmapping/Hitsound)
```
Comment (Walavouchey) [inline wiki/Music_theory/Rhythm/en.md]:
```suggestion
**Rhythm** is the pattern of sound, silence, and emphasis in a song. In music theory, rhythm refers to the recurrence of notes and rests (silences) in time. When a series of notes and rests repeats, it forms a rhythmic pattern. In addition to indicating when notes are played, musical rhythm also stipulates how long they are played and with what intensity. This creates different note durations and different types of accents. [[1]][r] In osu, notes are represented by [beats](/wiki/Music_theory/Beat), accents are represented by [hitsounds](/wiki/Beatmapping/Hitsound)
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Music_theory/Rhythm/en.md [hunk 12]
```diff
@@ -0,0 +1,11 @@
+---
+stub:true
+---
+
+# Rhythm
+
+**Rhythm** is the pattern of sound, silence, and emphasis in a song. In music theory, rhythm refers to the recurrence of notes and rests (silences) in time. When a series of notes and rests repeats, it forms a rhythmic pattern. In addition to indicating when notes are played, musical rhythm also stipulates how long they are played and with what intensity. This creates different note durations and different types of accents.[^1](r) In osu, notes are represented by [beats](/wiki/Music_theory/Beat), accents are represented by [hitsounds](/wiki/Beatmapping/Hitsound)
```
Comment (Walavouchey) [inline wiki/Music_theory/Rhythm/en.md:7]:
```suggestion
**Rhythm** is the pattern of sound, silence, and emphasis in a song. In music theory, rhythm refers to the recurrence of notes and rests (silences) in time. When a series of notes and rests repeats, it forms a rhythmic pattern. In addition to indicating when notes are played, musical rhythm also stipulates how long they are played and with what intensity. This creates different note durations and different types of accents. [[1]][r] In osu, notes are represented by [beats](/wiki/Music_theory/Beat), accents are represented by [hitsounds](/wiki/Beatmapping/Hitsound).
```

File: wiki/Music_theory/Syncopation/en.md [hunk 13]
```diff
@@ -0,0 +1,7 @@
+---
+stub: true
+---
+
+# Syncopation
+
+**Syncopation** is the accenting of a note which would usually not be accented. The [time signature](/wiki/Music_theory/Time_signature) of a piece of music gives an indication of a regular [rhythm](/wiki/Music_theory/Rhythm) pattern of strong and weak beats. A syncopated rhythm goes against this expectation by putting the accent on weak beats, thereby creating counterflow.
```
Comment (Walavouchey) [inline wiki/Music_theory/Syncopation/en.md:7]: You could additionally explain it more simply with something along the lines of "placing notes off-beat, i.e. on 'red timeline ticks'"

**Review by Walavouchey — CHANGES_REQUESTED** [review]
needs more review i suppose

File: wiki/Music_theory/Rhythm/en.md [hunk 14]
```diff
@@ -0,0 +1,13 @@
+---
+stub: true
+---
+
+# Rhythm
+
+**Rhythm** is the pattern of sound, silence, and emphasis in a song. In music theory, rhythm refers to the recurrence of notes and rests (silences) in time. When a series of notes and rests repeats, it forms a rhythmic pattern. In addition to indicating when notes are played, musical rhythm also stipulates how long they are played and with what intensity. This creates different note durations and different types of accents. [[1]][r] In osu, notes are represented by [beats](/wiki/Music_theory/Beat), accents are represented by [hitsounds](/wiki/Beatmapping/Hitsound).
```
[the remark this answers is not in the document]
Comment (Gamingzone123) [reply wiki/Music_theory/Rhythm/en.md]: ive done something with it, not sure if its sufficeint though

File: wiki/Music_theory/Syncopation/en.md (hunk 13 above)
Comment (Gamingzone123) [reply wiki/Music_theory/Syncopation/en.md]: added

**Comment by Walavouchey** [comment]
ah, no, by redirects i meant in `redirects.yaml`. i'll do those before merging

also please be a bit more careful when staging files, `wiki/.vs/slnx.sqlite` was committed in [0f85fbe](https://github.com/ppy/osu-wiki/pull/6995/commits/0f85fbefc03b7ef208429ae7ace49afcc15ff5c2)

**Comment by TicClick** [comment]
I'll post a review in a few days

File: wiki/Music_theory/Measure/en.md [hunk 15]
```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+tags:
+  - bar
+---
+
+# Measure
+
+A **measure** (or **bar**) in music theory refers to a single unit of time featuring a specific number of [beats](/wiki/Music_theory/Beat) played at a particular [tempo](/wiki/Music_theory/Tempo).
```
Comment (TicClick) [inline wiki/Music_theory/Measure/en.md:9]: I have a feeling that this page may be safely combined with the one about time signatures -- what do you think? like, is there any more material outside of the article that could be brought in to remove `stub: true`?

File: wiki/Music_theory/Measure/en.md [hunk 16]
```diff
@@ -0,0 +1,15 @@
...
+
+In the [beatmap editor timeline](/wiki/Client/Beatmap_editor/Timelines), a measure can be identified as the gap between two large white ticks:
+
+![A measure in the timing tab timeline](img/measure_timing.png "White ticks represent beats, with the larger ones representing the start of a new measure.")
```
Comment (TicClick) [inline wiki/Music_theory/Measure/en.md:13]: this will likely render poorly on the wiki. instead, I'd put the image after the title to illustrate the whole article, and make this sentence complete.

File: wiki/Music_theory/Rhythm/en.md [hunk 17]
```diff
@@ -0,0 +1,13 @@
+---
+stub: true
+---
+
+# Rhythm
+
+**Rhythm** is the pattern of sound, silence, and emphasis in a song.[[1]][r] In osu, mappers utilise these rythms when creating beatmaps, notes are represented by [beats](/wiki/Music_theory/Beat) and accents are represented by [hitsounds](/wiki/Beatmapping/Hitsound).
```
Comment (TicClick) [inline wiki/Music_theory/Rhythm/en.md:7]: other than that, it kind of works as a stub

```suggestion
**Rhythm** is the pattern of sound, silence, and emphasis in a song.[[1]][r] In osu!, mappers utilise these combinations when creating beatmaps; notes are represented by [beats](/wiki/Music_theory/Beat) and accents are represented by [hitsounds](/wiki/Beatmapping/Hitsound).
```

File: wiki/Music_theory/Syncopation/en.md [hunk 18]
```diff
@@ -0,0 +1,7 @@
+---
+stub: true
+---
+
+# Syncopation
+
+**Syncopation** is the accenting of a note which would usually not be accented. Syncopation can also be mapped in somewhat by placing notes off-beat i.e. red or blue timeline ticks. The [time signature](/wiki/Music_theory/Time_signature) of a piece of music gives an indication of a regular [rhythm](/wiki/Music_theory/Rhythm) pattern of strong and weak beats. A syncopated rhythm goes against this expectation by putting the accent on weak beats, thereby creating counterflow.
```
Comment (TicClick) [inline wiki/Music_theory/Syncopation/en.md:7]: this may need an example (even an image?) to better convey the topic

File: wiki/Music_theory/Time_signature/en.md [hunk 19]
```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+tags:
+  - music theory
+  - mapping
+  - beatmapping
+  - tips
```
Comment (TicClick) [inline wiki/Music_theory/Time_signature/en.md:7]: these tags have little relation to the article (and "music theory" is too general imo)

File: wiki/Music_theory/Time_signature/en.md [hunk 20]
```diff
@@ -0,0 +1,17 @@
...
+  - sheet music
+---
+
+# Time signature
+
+*For more information on time signatures in music theory, see: [Time signature](https://en.wikipedia.org/wiki/Time_signature)*
+
+A **time signature** is a musical term that indicates the number of [beats](/wiki/Beat) in a single measure (indicated by the top number), and which note indicates a single beat (indicated by the bottom number).
```
Comment (TicClick) [inline wiki/Music_theory/Time_signature/en.md:15]: - too much "indicate"s (I counted 4)
- you use terms such as "top number" and "bottom number", but before that there is nothing about what a time signature actually looks like

File: wiki/Music_theory/Time_signature/en.md [hunk 21]
```diff
@@ -0,0 +1,17 @@
...
+
+osu! offers two alternative time signatures for [mapping](/wiki/Beatmapping), which can be selected from the `Timing` menu in the [beatmap editor](/wiki/Client/Beatmap_editor). These include 4/4 (a.k.a. "common time" signature), and 3/4 (a.k.a. the "Waltz" or "three-four-time" signature). osu! will not prevent users from mapping songs that are written in different signatures, although it is not guaranteed that the beatmap will align with the song if this is done.
```
Comment (TicClick) [inline wiki/Music_theory/Time_signature/en.md:17]: can't really understand what the last sentence means:
- osu! also provides 5/4, 6/4, and 7/4 time signatures from the editor
- "it is not guaranteed that the beatmap will align with the song" -- it's a mapper's task to make sure a beatmap follows the song, did you maybe want to say something like "songs with other signatures are disappearingly rare"?

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Music_theory/Measure/en.md (hunk 15 above)
Comment (Gamingzone123) [reply wiki/Music_theory/Measure/en.md:9]: probably, i wouldn't know how to properly combine them though

File: wiki/Music_theory/Rhythm/en.md (hunk 17 above)
Comment (Gamingzone123) [reply wiki/Music_theory/Rhythm/en.md:7]: weren't we removing the exclamation or did i miss something?

File: wiki/Music_theory/Time_signature/en.md (hunk 19 above)
Comment (Gamingzone123) [reply wiki/Music_theory/Time_signature/en.md:7]: i didn't write the article but i agree lol

File: wiki/Music_theory/Time_signature/en.md (hunk 20 above)
Comment (Gamingzone123) [reply wiki/Music_theory/Time_signature/en.md:15]: i didnt write the article

File: wiki/Music_theory/Time_signature/en.md (hunk 21 above)
Comment (Gamingzone123) [reply wiki/Music_theory/Time_signature/en.md:17]: i didn't write the article, just moved it

File: wiki/Music_theory/Rhythm/en.md (hunk 17 above)
Comment (Walavouchey) [reply wiki/Music_theory/Rhythm/en.md:7]: what? no, it's the branding

File: wiki/Music_theory/Measure/en.md (hunk 16 above)
Comment (Gamingzone123) [reply wiki/Music_theory/Measure/en.md]: moved

File: wiki/Music_theory/Measure/en.md [hunk 22]
```diff
@@ -0,0 +1,15 @@
...
+
+![A measure in the compose tab timeline](img/measure_compose.png "White ticks represent beats, with the larger ones representing the start of a new measure")
```
[the remark this answers is not in the document]
Comment (Gamingzone123) [reply wiki/Music_theory/Measure/en.md]: removed

File: wiki/Music_theory/Syncopation/en.md (hunk 18 above)
Comment (Gamingzone123) [reply wiki/Music_theory/Syncopation/en.md:7]: ill mark as todo, as i dont really have an awful lot of time at the moment, a lot of stuff going on irl

File: wiki/Music_theory/Measure/en.md (hunk 22 above)
[the remark this answers is not in the document]
Comment (Gamingzone123) [reply wiki/Music_theory/Measure/en.md]: removed

**Comment by TicClick** [comment]
@Walavouchey think it's better if you will recheck my own changes and merge

File: wiki/Music_theory/Measure/en.md (hunk 15 above)
Comment (Walavouchey) [reply wiki/Music_theory/Measure/en.md:11]: to me this stub doesn't look too bad as is

File: wiki/Music_theory/Tempo/en.md [hunk 23]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Tempo
+
+**Tempo** is the speed at which a piece of music is played and is communicated, measured in beats per minute (BPM).
+
+[Screenshot of BPM on the beatmap information page](img/BPM_beatmap_info.png "The BPM of a song is shown on a beatmap's information page next to the metronome symbol")
```
Comment (Walavouchey) [inline wiki/Music_theory/Tempo/en.md:9]: i don't think this screenshot is really necessary - i'd rather expand the lead paragraph a bit instead

**Comment by Walavouchey** [comment]
@TicClick i've pushed some changes. do tell if something looks odd, otherwise this should be good to be merged

**Review by Walavouchey — APPROVED** [review]

**Review by TicClick — APPROVED** [review]
